Library/Mesh Model/Element.swift

```diff
@@ -234,7 +234,8 @@ internal extension Element {
         insert(model: Model(sigModelId: .configurationClientModelId,
                             delegate: ConfigurationClientHandler(meshNetwork)), at: 1)
         insert(model: Model(sigModelId: .healthServerModelId), at: 2)
-        insert(model: Model(sigModelId: .healthClientModelId), at: 3)
+        insert(model: Model(sigModelId: .healthClientModelId,
+                            delegate: HealthClientHandler()), at: 1)
```

Comment:
`at: 3`, not `at: 1`

Reply:
https://github.com/NordicSemiconductor/IOS-nRF-Mesh-Library/blob/fd8e5252e26bd62b2c746c3e5a35a1215edcc039/Library/Mesh%20Model/Element.swift#L237-L238